src/simsopt/geo/surface.py

```diff
@@ -1,4 +1,5 @@
 import abc
+from enum import Enum
```

Comment:
I don't see `Enum` used anywhere.

Reply:
It is the remnant of playing with Enum class for Surface.RANGE_FULL_TORUS... Will delete it.

Reply:
Deleted the enum import